### PR TITLE
ARTEMIS-1237 - Hard coded timeout on Journal file open

### PR DESCRIPTION
**Pull request by andytaylor**
Make timeout configurable

https://issues.apache.org/jira/browse/ARTEMIS-1237

### CODE REVIEW

File: artemis-server/src/main/resources/schema/artemis-configuration.xsd [hunk 1]
```diff
@@ -688,6 +688,15 @@
             </xsd:annotation>
          </xsd:element>
 
+         <xsd:element name="journal-file-open-timeout" type="xsd:int" maxOccurs="1" minOccurs="0">
+            <xsd:annotation>
+               <xsd:documentation>
+                  the maximum number of write requests that can be in the AIO queue at any one time. Default is 500 for
```
Comment (michaelandrepearce) [inline artemis-server/src/main/resources/schema/artemis-configuration.xsd]: documentation doesn't seem to match element. Assume it should read more like.

> the length of time to wait when opening a new Journal file before timing out and failing

Comment (andytaylor) [inline artemis-server/src/main/resources/schema/artemis-configuration.xsd]: fixed

Comment (michaelandrepearce) [inline artemis-server/src/main/resources/schema/artemis-configuration.xsd]: Thanks :)

**Comment by clebertsuconic** [comment]
@andytaylor  where is master's PR?

**Comment by clebertsuconic** [comment]
@andytaylor we can't make a change on 1.x only... it needs to be on master...

I tried cherry-pick but had tons of conflicts.. can you do it?

**Comment by andytaylor** [comment]
https://github.com/apache/activemq-artemis/pull/1359

**Comment by clebertsuconic** [comment]
can you close this please? merged with a rebase! apache bot won't close 1.x

**Pull request closed without merging**